.azure-pipelines/statusChain.py

```diff
@@ -0,0 +1,58 @@
+import time
+
+TestPlanStatus = {
+    "INIT": 10,
+    "LOCK_TESTBED": 20,
+    "PREPARE_TESTBED": 30,
+    "EXECUTING": 40,
+    "KVMDUMP": 50,
+    "FAILED": 60,
+    "CANCELLED": 70,
+    "FINISHED": 80
+}
+
+
+class AbstractStatus():
+    def __init__(self, status):
+        self.status = status
+
+    def print_logs(self, test_plan_id, resp_data, start_time):
+        status = resp_data.get("status", None)
+        if(TestPlanStatus[status] == self.status):
+            print("Test plan id: {}, status: {},  elapsed: {:.0f} seconds"
+                  .format(test_plan_id, resp_data.get("status", None), time.time() - start_time))
+
+
+class InitStatus(AbstractStatus):
+    pass
```

Comment:
you can set the status during the initialization,
and no need to pass the status as a parameter:

    def __init__(self):
        super().__init__(TestPlanStatus.INIT)